src/OSes/macos.py

```diff
@@ -25,29 +26,29 @@ def setup(tld=config.TOP_LEVEL_DOMAIN):
     open(PLIST_PATH, 'w').write(plist)
     os.system(f'sudo launchctl load -w {PLIST_PATH} 1>/dev/null 2>/dev/null')
 
-    return True
+    return 0
```

Comment:
Aqui o retorno anterior era true. Se for retornar int 0 ou 1, não seria melhor retornar 1?

Reply:
status 0 é sucesso. Qualquer coisa diferente de 0 seria um erro.